JSDemos/Demos/PivotGrid/SimpleArray/description.md

```diff
@@ -1 +1,5 @@
-The PivotGrid is&nbsp;a&nbsp;high-performance, enterprise-ready UI component optimized for multi-dimensional data analysis. It&nbsp;allows your end users to&nbsp;easily arrange and summarize information in&nbsp;a&nbsp;cross-tabular format. In&nbsp;this demo, the PivotGrid&rsquo;s simplicity of&nbsp;use is&nbsp;illustrated. Sorting and filtering are enabled so&nbsp;you can sort data in&nbsp;ascending or&nbsp;descending order and by&nbsp;summary values. You can drag and drop fields to&nbsp;reorganize the PivotGrid and change filtering properties to&nbsp;analyze data more efficiently.
\ No newline at end of file
+You can use the PivotGrid component to display and edit data from an array of objects. Use the [dataSource](/Documentation/ApiReference/UI_Components/dxPivotGrid/Configuration/#dataSource) property to bind the PivotGrid to data. The [store](/Documentation/ApiReference/Data_Layer/DataSource/Configuration/store/) property of the **dataSource** specifies the bound array.
+
+Use the [fields[]](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/) property to specify fields to be displayed as pivot grid cells. This property accepts an array of objects whose fields contain cell settings depending on the cell [area](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/#area). Most cell types require that you specify the [dataField](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/#dataField) property to specify which field provides data for a cell.
```

Comment:
The paragraph lost my attention at the second sentence. Try to simplify.

Also, you're talking about cells but I don't think that you use correct terminology. There are *pivot grid fields* that get data from *data fields*. Pivot grid fields can be placed into *areas*, in which case field values are displayed. Field values from the column and row areas are displayed in *column and row headers*. Field values from the data area are called *summary values*.

The PivotGrid terminology may be confusing indeed. I can only suggest reading the [fields](https://js.devexpress.com/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/) API reference straight through to better understand what's what.